src/client/components/WeekWithDetail/index.js

```diff
@@ -89,20 +96,24 @@ function WeekWithDetail(props) {
           values={{ ...dates, weekForUser }}
         />
       }
-      expandedBody={
+      content={
         <WeekConfirmationDetails
           employers={weekHasEmployers ? weekData.employers : undefined}
           questionAnswers={questionAnswers}
           questionKeys={questionKeys}
           weekString={toWeekString(weekIndex)}
         />
       }
+      showContent={showContent[index]}
+      toggleContent={() => toggleContent(index)}
     />
   );
 }
 
 WeekWithDetail.propTypes = {
   index: PropTypes.number.isRequired,
+  showContent: PropTypes.arrayOf(PropTypes.bool).isRequired,
```

Comment:
Same here, maybe call this showContents or showContentsArray to differentiate it from showContent in AccordianItem.